id_1000_random/creating_tasks/bigquery_credentials.dat

```diff
@@ -0,0 +1 @@
+{"_module": "oauth2client.client", "scopes": ["https://www.googleapis.com/auth/bigquery"], "token_expiry": "2017-01-06T18:39:41Z", "id_token": null, "access_token": "ya29.Ci_LA_XiKSDQRFt5QQmDEN2xL4whMPTk1hxbTqfRhFUUFX-2ofCMEUF8gPaKhZYWZQ", "token_uri": "https://www.googleapis.com/oauth2/v4/token", "invalid": false, "token_response": {"access_token": "ya29.Ci_LA_XiKSDQRFt5QQmDEN2xL4whMPTk1hxbTqfRhFUUFX-2ofCMEUF8gPaKhZYWZQ", "token_type": "Bearer", "expires_in": 3600, "refresh_token": "1/NtmSOHUxdwtZgGO-N-7IAmyH-lh7hGHYRgHY162ZJog"}, "client_id": "495642085510-k0tmvj2m941jhre2nbqka17vqpjfddtd.apps.googleusercontent.com", "token_info_uri": "https://www.googleapis.com/oauth2/v3/tokeninfo", "client_secret": "kOc9wMptUtxkcIFbtZCcrEAc", "revoke_uri": "https://accounts.google.com/o/oauth2/revoke", "_class": "OAuth2Credentials", "refresh_token": "1/NtmSOHUxdwtZgGO-N-7IAmyH-lh7hGHYRgHY162ZJog", "user_agent": null}
```

Comment:
It seems like a bad idea to put big_query credentials in the repo. On the other hand, looks like they might be
expired, but then it still seems like a bad idea to put them in the repo :-)